src/biomappings/wsgi.py

```diff
@@ -241,6 +241,8 @@ def get_curie(prefix: str, identifier: str) -> str:
     def get_url(cls, prefix: str, identifier: str) -> str:
         """Return URL for a given prefix and identifier."""
         if bioregistry.get_obofoundry_prefix(prefix):
+            if prefix == 'hp' and identifier.startswith('HP:'):
+                identifier = identifier[3:]
```

Comment:
@cthoyt this is a local patch to make links work here, I suspect the issue could be more general and have to do with namespaces embedded in IDs. Is there a more general solution for this?

Reply:
hmm more generally I guess we need to remove bananas here. I will check in the `bioregistry.get_ols_iri` to see why it doesn't do that automatically.

Reply:
I added a generic solution for this in 54c2f2a